packages/flutter/lib/src/cupertino/colors.dart

```diff
@@ -3,6 +3,19 @@
 // found in the LICENSE file.
 
 import 'dart:ui' show Color;
+import 'package:collection/collection.dart' show DeepCollectionEquality;
+
+import '../../foundation.dart';
+import '../widgets/basic.dart';
+import '../widgets/framework.dart';
+import '../widgets/media_query.dart';
+import 'interface_level.dart';
+import 'theme.dart';
+
+// Examples can assume:
+// Widget child;
+// Color lightModeColor;
+// Color darkModeColor;
 
 /// A palette of [Color] constants that describe colors commonly used when
 /// matching the iOS platform aesthetics.
```

Comment:
Now is probably not the best time, since we're still using the fallback values in `CupertinoSystemColors`, and we're not even using those new colors ourselves?  Maybe as the last step of dark mode?